opentech/apply/funds/templates/funds/applicationsubmission_simplified_detail.html

```diff
@@ -0,0 +1,58 @@
+{% extends "base-apply.html" %}
+
+{% load wagtailcore_tags statusbar_tags %}
+
+{% block title %}{{ object.title }}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block content %}
+{% if object.round.specific.is_sealed %}
+<ul class="messages">
+    {% include "includes/message_item.html" with message="This submission is sealed " tag="warning" close=False %}
+</ul>
+{% endif %}
+<div class="admin-bar">
+    <div class="admin-bar__inner">
+        <h1 class="beta heading heading--no-margin heading--bold">{{ object.title }}</h1>
+        <h5 class="heading heading--meta">
+            <span>{{ object.stage }}</span>
+            <span>{% if public_page %}<a class="link--transparent link--underlined" href="{% pageurl public_page %}" >{{ object.page }}</a>{% else %}{{ object.page }}{% endif %}</span>
+            {% if request.user.is_apply_staff and object.round %}
```

Comment:
Can we clear the logic here - this is meant to be a simple view aligned with what compliance will be seeing so no links out.

Reply:
Fixed and rebased.